### PR TITLE
Add moderators to listing

### PR DESCRIPTION
**Pull request by placer14**

### CODE REVIEW

**Comment by jjeffryes** [comment]
@placer14 this is working as expected when I use it with my verified moderators branch.

**Comment by cpacia** [comment]
`RepoVersion` in repo/init.go will need to be bumped to 7.

File: repo/init.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	"time"
 )
 
-const RepoVersion = "6"
+const RepoVersion = "7"
```
Comment (placer14) [inline repo/init.go]: Does this value represent the desired migration version? Is there any reason we couldn't use the filenames of the migrations to determine this?

Comment (cpacia) [reply repo/init.go]: This is just what the file is set to when initializing a new node. I suppose you could use the migration  index+1 for it.

**Comment by placer14** [comment]
@cpacia I'm happy to have this merged when you're ready. Thanks for the assist!

File: repo/init.go (hunk 1 above)
Comment (placer14) [reply repo/init.go]: I'll make an issue for this to track for later fixing.

**Pull request merged**